dockhand-lru-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "dockhand-lru-registry.name" . }}
+  labels:
+    {{- include "dockhand-lru-registry.labels" . | nindent 4 }}
+spec:
+  serviceName: {{ include "dockhand-lru-registry.name" . }}
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "dockhand-lru-registry.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "dockhand-lru-registry.selectorLabels" . | nindent 8 }}
+      annotations:
+        checksum/config-maps: {{ (.Files.Glob "files/**").AsConfig | sha256sum }}
+    spec:
```

Comment:
should we add a default `podSecurityPolicy` to run as a non-root user?  or at least provide options for the user to supply a `podSecurityPolicy`?

looks like `distribution/distribution` does default to `root`, and getting the permissions right on a newly minted Persistent Volume may be non-trivial with a non-root user.  so it might be best to leave the default as `root` but still provide an option to the user to specify a non-root policy

Reply:
We could... but atm both the `boxboat/dockhand-lru-registry` and the `registry` image assume `root`.

Reply:
Added `securityContext` settings